### PR TITLE
Adding CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by tanvimoharir**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
```
Comment (Pranav016) [inline CODE_OF_CONDUCT.md:71]: Turn the link to a hyper link just like you did to the email.
Open links don't look good.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see
+https://www.contributor-covenant.org/faq
```
Comment (Pranav016) [inline CODE_OF_CONDUCT.md:76]: Make this a hyper link.
Example
```
[Contributor Convent FAQ](https://www.contributor-covenant.org/faq)
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
```
Comment (Pranav016) [inline CODE_OF_CONDUCT.md:44]: Turn these 2 into bullet points.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
A couple of changes required to remove exposed links.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (tanvimoharir) [reply CODE_OF_CONDUCT.md]: Changed.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (tanvimoharir) [reply CODE_OF_CONDUCT.md]: This is done.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (tanvimoharir) [reply CODE_OF_CONDUCT.md]: This is done.

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**